leshan-tl-javacoap-server/pom.xml

```
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+Copyright (c) 2013-2015 Sierra Wireless and others.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v2.0
+and Eclipse Distribution License v1.0 which accompany this distribution.
+
+The Eclipse Public License is available at
+   http://www.eclipse.org/legal/epl-v20.html
+and the Eclipse Distribution License is available at
+   http://www.eclipse.org/org/documents/edl-v10.html.
+
+Contributors:
+    Sierra Wireless - initial API and implementation
+    Bosch Software Innovations GmbH - OSGi support
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.leshan</groupId>
+    <artifactId>lib-build-config</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+    <relativePath>../build-config/lib-build-config/pom.xml</relativePath>
+  </parent>
+  <artifactId>leshan-tl-javacoap-server</artifactId>
+  <packaging>bundle</packaging>
+  <name>leshan - server java-coap</name>
+  <description>A transport implementation for leshan server based on Java CoAP</description>
+
+	<repositories>
+	    <repository>
+	        <id>jitpack.io</id>
+	        <url>https://jitpack.io</url>
+	    </repository>
+	</repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.leshan</groupId>
+      <artifactId>leshan-server-core</artifactId>
+    </dependency>
+    
+    <dependency>
+    <groupId>com.github.open-coap.java-coap</groupId>
+    <artifactId>coap-core</artifactId>
+    <version>6.5.0</version>
```

You can update it, it is now in central, jitpack was not very reliable.
```suggestion
    <groupId>io.github.open-coap</groupId>
    <artifactId>coap-core</artifactId>
    <version>6.8.0</version>
```

---

:tada: **Available on maven central ! :tada:** 

I changed it. (0fbcfb3)

But updating I find a kind of API break between 6.5 and 6.8

`DataConvertingUtility.parseUriQuery(String)` was raising `ParseException` and now it doesn't.
Note that `DataConvertingUtility.parseUriQueryMult(String)` still raises a `ParseException`
I don't know if this is expected :shrug: 

I also updated comparison table at  https://github.com/eclipse/leshan/issues/1338.

---

Yes, that was expected.

---

Yes, that was expected.